### PR TITLE
Update reference to the size of PyPI

### PR DESCRIPTION
**Pull request by irismessage**
Last changed in 2008 (528576031d9655ca8004260a2bf2c783f77d1da1).

I also changed "growing collection" to "constantly growing collection" since it makes more sense to me with the much larger number.

### CODE REVIEW

**Review by evieluvsrainbows — APPROVED** [review]

File: Doc/library/index.rst [hunk 1]
```diff
@@ -27,8 +27,8 @@ as a collection of packages, so it may be necessary to use the packaging
 tools provided with the operating system to obtain some or all of the
 optional components.
 
-In addition to the standard library, there is a growing collection of
-several thousand components (from individual programs and modules to
+In addition to the standard library, there is a constantly growing collection of
```
Comment (gpshead) [inline Doc/library/index.rst]: instead of constantly growing or growing, i'd go with something such as

"there is an evolving collection of"

Comment (irismessage) [reply Doc/library/index.rst]: how about 'active collection of'?

Comment (gpshead) [reply Doc/library/index.rst]: that also sounds good.

Comment (irismessage) [reply Doc/library/index.rst]: a57f3a1ceddb7187d57b1cd0bc7fa1c2454925a0

**Review by gpshead — APPROVED** [review]

**Pull request merged**